Packs/OpsGenie/Integrations/OpsGenieV3/OpsGenieV3.yml

```diff
@@ -670,6 +647,34 @@ script:
       description: Whether the request was successful.
       type: Boolean
 
+  - arguments:
+    - default: false
+      description: E-mail address of the user
+      isArray: false
+      name: username
+      required: false
```

Comment:
In the README file it has the required value, as I comment you there it should be true or false but if it's True then it should be here too. (Also for the other arguments)